template-sample-app/template/cypress.json

```diff
@@ -1 +1,3 @@
-{}
+{
+  "defaultCommandTimeout": 10000
```

Comment:
1e4? It's just a personal preference :+1:

Reply:
I usually find scientific notation to be a bit less clear for most people